app/models/request.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+# Explicitly require the file from your model.
+require_relative "../operations/project_create"
```

Comment:
I'd love to avoid one-off require statements if we can. These cause confusing bugs down the road. I would think that the operations folder, by virtue of being inside of the app folder, would just be loaded? Or maybe there is a load_path that needs adjusting?